### PR TITLE
Rename test_model_common_attributes -> test_model_get_set_embeddings

### PR DESCRIPTION
**Pull request by amyeroberts**
# What does this PR do?

The method name is misleading - it is testing being able to get and set embeddings, not common attributes to all models

### CODE REVIEW

File: tests/test_modeling_common.py [hunk 1]
```diff
@@ -1971,13 +1971,17 @@ def test_resize_embeddings_untied(self):
...
-    def test_model_common_attributes(self):
+    def test_model_get_set_embeddings(self):
         config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
 
         for model_class in self.all_model_classes:
             model = model_class(config)
             self.assertIsInstance(model.get_input_embeddings(), (nn.Embedding, AdaptiveEmbedding))
-            model.set_input_embeddings(nn.Embedding(10, 10))
+
+            new_input_embedding_layer = nn.Embedding(10, 10)
```
Comment (ydshieh) [inline tests/test_modeling_common.py:1981]: a nit question: what if the previous line gives `AdaptiveEmbedding`? Do we even care this case?

**Review by ydshieh — APPROVED** [review]
Other than a nit question, the change makes a lot of sense. Thank you for the updating.

File: tests/test_modeling_common.py (hunk 1 above)
Comment (amyeroberts) [reply tests/test_modeling_common.py:1981]: I don't think we care, we just want to make sure we can set new ones

Comment (ydshieh) [reply tests/test_modeling_common.py:1981]: Understand and agree!

**Pull request merged**